tox.ini

```diff
@@ -0,0 +1,7 @@
+[tox]
+envlist = py27, py35
+
+[testenv]
+commands = nosetests
+deps =
+    -rrequirements.txt
```

Comment:
Oh sorry just noticed I think this is a typo with two r's at the start. Otherwise LGTM 👍

Reply:
This is actually the real syntax ! https://tox.readthedocs.io/en/latest/example/basic.html#depending-on-requirements-txt-or-defining-constraints
Bit weird I agree 🤷‍♀️